src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrdersController.cs

```diff
@@ -24,28 +24,29 @@ public sealed class OrdersController : Controller
     {
         private readonly IOrderRepository _orderRepository;
         private readonly ICreateOrderService _createOrderService;
+        private readonly IServiceRecipientRepository _serviceRecipientRepository;
```

Comment:
I know you havent changed this in this PR, but is it worth changing the ":Controller" to ":ControllerBase", which most of the other controllers are now using. Then you can get rid of the "using" keyword in the tests

Reply:
👍